docs/application/web/api/toc.xml

```diff
@@ -5767,6 +5770,9 @@
 						<topic href="html/ui_fw_api/Wearable_UIComponents/wearable_toggleswitch.htm" label="ToggleSwitch"></topic>
 						<topic href="html/ui_fw_api/Wearable_UIComponents/wearable_viewSwitcher.htm" label="ViewSwitcher"></topic>
 						<topic href="html/ui_fw_api/Wearable_UIComponents/wearable_virtuallist.htm" label="VirtualList"></topic>
+						<topic href="html/ui_fw_api/Wearable_UIComponents/wearable_numberpicker.htm" label="NumberPicker"></topic>
```

Comment:
It looks like these files are to be added to 4.0 and 3.0 also, according to this toc.xml update.
(Actually for 3.0, I can't sure as 3.0 is not commented in description.)
> [Problem] Thera are some missing pages for TAU 4.0/5.0 documentation

However as now the files are only added at 5.0,
would you please add `wearable_numberpicker.htm`, `wearable_timepicker.htm` and `wearable_datepicker.htm` files under desired folders and update their index.htm & wearable_component_list.htm file too?
Under 
   - /docs/application/web/api/3.0/ui_fw_api/Wearable_UIComponents/
   - /docs/application/web/api/4.0/ui_fw_api/Wearable_UIComponents/

Also need fix for
 - /docs/application/web/api/3.0/index.htm
 - /docs/application/web/api/3.0/ui_fw_api/Wearable_UIComponents/wearable_component_list.htm
 - /docs/application/web/api/4.0/index.htm
 - /docs/application/web/api/4.0/ui_fw_api/Wearable_UIComponents/wearable_component_list.htm

If we don't need this adding to 3.0, you don't need fix for 3.0.

Reply:
@lslachciak

Reply:
This change shouldn't be applied for 3.0

Reply:
I remove links to date picker, number picker and time picker in file docs/application/web/api/toc.xml for 3.0 and 4.0.